src/schemas/chat.py

```
@@ -15,8 +15,8 @@
 - ChatResult: CommonResponse의 result 필드에 들어가는 응답 데이터
 """
 
-from pydantic import BaseModel
-from typing import List
+from pydantic import BaseModel, field_validator
+from typing import List, Optional
```

Ruff (0.15.4)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)